### DISCUSSION TITLE
Calendar widget and day background color

### DISCUSSION THREAD

**Discussion by mschlindwein** [forus-labs/forui#289] [Q&A] [answered]
Hello, I just have a question to the calendar widget: is it possible to change the background color of days, where there is an existing appointment (query from the backend)? Or to add a small "dot" underneath a day, where there is an existing appointment?
Thank you!
Regards
Martin

**Comment by Pante** [comment]
It depends on what the expected behavior is when selecting a date with an existing appointment.

If a date with an existing appointment cannot be selected, you can make those dates unselectable (and optionally customize how unselectable dates look like, https://pub.dev/documentation/forui/latest/forui.widgets.calendar/FCalendarDayStyle-class.html). 

Unfortunately, we currently don't allow selectable dates to have various styles/customize the contents of a selectable date.

**Comment by Pante** [comment, marked as answer]
We've added `FCalendar.dayBuilder` which should allow greater flexibility when customizing the calendar. It should be available in 0.8.0.
